src/styles/stopwatch.scss

```diff
@@ -0,0 +1,53 @@
+.stopwatch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 550px;
+  height: 550px;
+  border: 2px solid rgb(204, 72, 171);
+  border-radius: 50%;
+  outline: 2px solid rgb(82, 90, 82);
+  outline-offset: 4px;
+
+  &__circle {
+    position: relative;
+    width: 30px;
+    height: 30px;
+    background-color: rgb(204, 72, 171);
+    border-radius: 50%;
+  }
+
+  &__minute {
+    position: absolute;
+    top: 57px;
+    z-index: -1;
+    width: 10px;
+    height: 250px;
```

Comment:
you use some magic numbers, check point 10 in checklist, will be better if you'll have the variables for main values that should contain sizes for hands and main indents